aspnetcore/performance/caching/distributed/samples/2.x/DistCacheSample/SampleApp.csproj

```diff
@@ -6,7 +6,6 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.App" />
```

Comment:
Last I heard from @natemcmaster is we're recommending an explicit version now. Let's make that change here.

Reply:
Whaaaaaa ... again?

https://github.com/aspnet/Docs/issues/8579#issuecomment-424548306
https://github.com/dotnet/sdk/pull/2533

Reply:
@guardrex is correct. We're sticking with the implicit PackageRef version for 2.x. Although we temporarily removed our guidance about using implicit versions, we could not land on a satisfactory substitute for the 2.x timeframe. So, we restored guidance to use the versionless reference in https://github.com/aspnet/Docs/pull/8707.

FWIW we're refactoring all of this in 3.0 so there won't be a PackageReference to Microsoft.AspNetCore.App at all.